### PR TITLE
docs: updates for 1.8 & 1.9

### PR DESCRIPTION
**Pull request by devcatalin**
This PR...

## Changes

-

## Fixes

-

## How to test it

-

## Screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/compare-sync.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Compare & Sync
+
+The Compare & Sync functionality allows you to compare and synchronize two sets of resources.
+
+Each set of resources can be one of the following:
+- Resources from your local manifest files
+- Resources from a cluster
```
Comment (jfermi) [inline docs/compare-sync.md:7]:
```suggestion
- Resources from a cluster.
```

File: docs/helm.md [hunk 2]
```diff
@@ -86,9 +86,30 @@ Monokle is now in **Preview Mode** (as indicated by the header at the top):
 - Selecting **Preview** for a different values file will switch the preview to the output of Helm for that file.
 - Selecting **Exit** in the top right or next to the values file restores the previous resource navigator.
 
-## **Editing of Values Files During Preview**
+## **Editing of Values Files**
+
+If you're editing Helm Values files, Monokle will highlight all properties that have been referenced by any templates.
+The list of paths to template files is shown when hovering the highlighted value.
```
Comment (jfermi) [inline docs/helm.md:92]:
```suggestion
The list of paths to template files is shown when hovering over the highlighted value.
```

File: docs/helm.md [hunk 3]
```diff
@@ -86,9 +86,30 @@ Monokle is now in **Preview Mode** (as indicated by the header at the top):
...
 you to quickly assess the impact of any changes on the generated files, instead of having to exit and recreate the preview to make a change.
  
 ![Edit Helm](img/helm-preview-reload-1.5.0.png)
+
+## **Working with Helm Templates**
+
+Each Helm Chart from the Helm pane has a subsection which displays all templates of that chart.
+Clicking on a template will select it and open it in the Editor.
+
+When you hover a value referenced from a Values file, the popup that appears will display the following information:
```
Comment (jfermi) [inline docs/helm.md:106]:
```suggestion
When you hover over a value referenced from a Values file, the popup that appears will display the following information:
```

File: docs/helm.md [hunk 4]
```diff
@@ -86,9 +86,30 @@ Monokle is now in **Preview Mode** (as indicated by the header at the top):
...
+- The path to the values file where the reference is from
```
Comment (jfermi) [inline docs/helm.md:107]:
```suggestion
- The path to the values file where the reference is from.
```

File: docs/helm.md [hunk 5]
```diff
@@ -86,9 +86,30 @@ Monokle is now in **Preview Mode** (as indicated by the header at the top):
...
+- The refereced value so you don't have to switch to the Values file
```
Comment (jfermi) [inline docs/helm.md:108]:
```suggestion
- The referenced value, so you don't have to switch to the Values file.
```

File: docs/helm.md [hunk 6]
```diff
@@ -86,9 +86,30 @@ Monokle is now in **Preview Mode** (as indicated by the header at the top):
...
+- A link to quickly navigate to the location of that value
```
Comment (jfermi) [inline docs/helm.md:109]:
```suggestion
- A link to quickly navigate to the location of that value.
```

File: docs/images.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
+# Working with Images
+
+Monokle has a dedicated Images panel which can be accessed from the left menu:
+
+![Images Panel](img/images-panel-1-8-0.png)
+
+This panel will show the list of all images that were found in the Navigator.
```
Comment (jfermi) [inline docs/images.md:7]:
```suggestion
This panel will show the list of all images that are found in the Navigator.
```

File: docs/images.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+The list of images will changed based on the Mode you're in:
+- Local mode
+- Preview mode (Helm or Kustomize)
+- Cluster mode
+
+Clicking on an Image will select it and will open an Image Details panel on top of the Editor.
+
+In the panel, you will find basic information about the image and you can browse the tags.
```
Comment (jfermi) [inline docs/images.md:15]:
```suggestion
In the panel, you will find basic information about the image and browse the image's tags.
```

File: docs/images.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+
+Hovering over an image will show the "Replace" action:
+
+![Image Replace Hover](img/images-panel-hover-1-8-0.png)
+
+Clicking on it will open a modal which allow you to enter a new tag for that image:
+
+![Image Replace Tag](img/images-panel-replace-1-8-0.png)
+
+After clicking on "Done", the tag will be replaced to the newly specified value across all your resources.
```
Comment (jfermi) [inline docs/images.md:25]:
```suggestion
After clicking "Done", the tag will be replaced with the newly specified value across all your resources.
```

File: docs/compare-sync.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+- Resources from the [Preview](helm.md) of a Helm Chart
```
Comment (jfermi) [inline docs/compare-sync.md:8]:
```suggestion
- Resources from the [Preview](helm.md) of a Helm Chart.
```

File: docs/compare-sync.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+- Resources from the [Preview](kustomize.md) of a Kustomization
```
Comment (jfermi) [inline docs/compare-sync.md:9]:
```suggestion
- Resources from the [Preview](kustomize.md) of a Kustomization.
```

File: docs/compare-sync.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+
+Here are some of the scenarios where Compare & Sync can help you:
+1. Compare local resources to the resources in your cluster AND:
+  - Deploy local resources to the cluster
```
Comment (jfermi) [inline docs/compare-sync.md:13]:
```suggestion
  - Deploy local resources to the cluster.
```

File: docs/compare-sync.md [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+  - Extract cluster resources to local
+2. Compare resources between two clusters AND:
+  - Deploy resources from one cluster to another
```
Comment (jfermi) [inline docs/compare-sync.md:16]:
```suggestion
  - Deploy resources from one cluster to another.
```

File: docs/compare-sync.md [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+3. Compare resources between a Helm Chart or Kustomization preview to a cluster AND:
+  - Deploy the output of the helm chart to the cluster
+4. Compare resources between a Helm Chart or Kustomization preview to your local resources AND:
+  - Extract resources from the preview to your local files
```
Comment (jfermi) [inline docs/compare-sync.md:20]:
```suggestion
  - Extract resources from the preview to your local files.
```

File: docs/compare-sync.md [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+5. Compare the output of a Helm Chart using two different values files
```
Comment (jfermi) [inline docs/compare-sync.md:21]:
```suggestion
5. Compare the output of a Helm Chart using two different values files.
```

File: docs/compare-sync.md [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+6. Compare the output of two different Helm Charts
```
Comment (jfermi) [inline docs/compare-sync.md:22]:
```suggestion
6. Compare the output of two different Helm Charts.
```

File: docs/compare-sync.md [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+7. Compare the output of a Helm Chart to the output of a Kustomization
```
Comment (jfermi) [inline docs/compare-sync.md:23]:
```suggestion
7. Compare the output of a Helm Chart to the output of a Kustomization.
```

File: docs/compare-sync.md [hunk 18]
```diff
@@ -0,0 +1,43 @@
...
+8. Compare two Kustomization outputs
+
+To use Compare & Sync follow these steps:
+
+- Browse to a folder containing manifests / kustomizations / Helm charts.
```
Comment (jfermi) [inline docs/compare-sync.md:28]:
```suggestion
- Browse to a folder containing manifests/Kustomizations/Helm charts.
```

File: docs/images.md [hunk 19]
```diff
@@ -0,0 +1,25 @@
...
+- Preview mode (Helm or Kustomize)
+- Cluster mode
+
+Clicking on an Image will select it and will open an Image Details panel on top of the Editor.
+
+In the panel, you will find basic information about the image and browse the image's tags.
+
+Hovering over an image will show the "Replace" action:
+
+![Image Replace Hover](img/images-panel-hover-1-8-0.png)
```
Comment (jfermi) [inline docs/images.md:19]: This image has a red line on top of the word 'Replace'.

**Review by jfermi — COMMENTED** [review]
One minor suggestion.

File: docs/compare-sync.md [hunk 20]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
 ![Compare & Sync](img/cluster-compare-button-1.6.0.png)
 
-The Compare & Sync modal opens and shows the local resources on the left and the cluster resources on the right. 
+In the image below you can see an overview of the Compare & Sync modal with each section highligthed in color which will be explained:
 
-TODO: Add overview image
+![Compare & Sync Overview](img/compare-sync-overview-1-9-1.png)
 
-TODO: How to configure each section with images
+The top bar highlited in red contains a checkbox to select all rows and then 4 numbered inputs:
```
Comment (jfermi) [inline docs/compare-sync.md:37]:
```suggestion
The top bar, highlighted in red, contains a checkbox to select all rows and then 4 numbered inputs:
```

File: docs/compare-sync.md [hunk 21]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
+1. Search input for quickly filtering all rows by the name of the resources
```
Comment (jfermi) [inline docs/compare-sync.md:38]:
```suggestion
1. Search input for quickly filtering all rows by the name of the resources.
```

File: docs/compare-sync.md [hunk 22]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
+2. Input for setting the default namespace that a resource should have if it's not already specified in it's yaml content
```
Comment (jfermi) [inline docs/compare-sync.md:39]:
```suggestion
2. Input for setting the default namespace that a resource should have if it's not already specified in it's yaml content.
```

File: docs/compare-sync.md [hunk 23]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
+3. Dropdown for selecting different options of comparison:
+  - `View all` shows all resources from both left and right columns
```
Comment (jfermi) [inline docs/compare-sync.md:41]:
```suggestion
  - `View all` shows all resources from both left and right columns.
```

File: docs/compare-sync.md [hunk 24]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
+  - `Only matching` will show only the resources that exist in both columns
```
Comment (jfermi) [inline docs/compare-sync.md:42]:
```suggestion
  - `Only matching` will show only the resources that exist in both columns.
```

File: docs/compare-sync.md [hunk 25]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
+  - `Only non-matching` will show resources that do not exist in either of the columns
```
Comment (jfermi) [inline docs/compare-sync.md:43]:
```suggestion
  - `Only non-matching` will show resources that do not exist in either of the columns.
```

File: docs/compare-sync.md [hunk 26]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
+  - `View left join` will show all resources from the left column, but only matching resources in the right column
```
Comment (jfermi) [inline docs/compare-sync.md:44]:
```suggestion
  - `View left join` will show all resources from the left column, but only matching resources in the right column.
```

File: docs/compare-sync.md [hunk 27]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
+  - `View right join` will show all resources from the right column, but only matching resources in the left column
```
Comment (jfermi) [inline docs/compare-sync.md:45]:
```suggestion
  - `View right join` will show all resources from the right column, but only matching resources in the left column.
```

File: docs/compare-sync.md [hunk 28]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
+4. Filter button which opens a panel that contains more filtering options
```
Comment (jfermi) [inline docs/compare-sync.md:46]:
```suggestion
4. Filter button which opens a panel that contains more filtering options.
```

File: docs/compare-sync.md [hunk 29]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
+Before configuring the columns, the UI will look like this:
 
-TODO: Filtering resources
+![Configure Compare & Sync](img/compare-sync-configure-1-9-1.png)
 
-TODO: Specifying "default" namespace for resources that don't have it specified
+Each column can be configured to specify the source of the resources that will be loaded in that column.
+If we look back at the Overview image from above, the green highlights show the following numbered inputs:
+1. The dropdown to choose the source type for that column. The options are:
+  - `Local` for loading the resources from the local files in the current project
```
Comment (jfermi) [inline docs/compare-sync.md:57]:
```suggestion
  - `Local` for loading the resources from the local files in the current project.
```

File: docs/compare-sync.md [hunk 30]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
+  - `Cluster` for loading resources from a cluster
```
Comment (jfermi) [inline docs/compare-sync.md:58]:
```suggestion
  - `Cluster` for loading resources from a cluster.
```

File: docs/compare-sync.md [hunk 31]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
+  - `Helm Preview` for loading the output of a Helm Chart dry run
```
Comment (jfermi) [inline docs/compare-sync.md:59]:
```suggestion
  - `Helm Preview` for loading the output of a Helm Chart dry run.
```

File: docs/compare-sync.md [hunk 32]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
+  - `Kustomize Preview` for loading the output of a Kustomization dry run
```
Comment (jfermi) [inline docs/compare-sync.md:60]:
```suggestion
  - `Kustomize Preview` for loading the output of a Kustomization dry run.
```

File: docs/compare-sync.md [hunk 33]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
+2. Button for reloading the current resource set
```
Comment (jfermi) [inline docs/compare-sync.md:61]:
```suggestion
2. Button for reloading the current resource set.
```

File: docs/compare-sync.md [hunk 34]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
+3. Button for clearing the configuration of that column
```
Comment (jfermi) [inline docs/compare-sync.md:62]:
```suggestion
3. Button for clearing the configuration of that column.
```

File: docs/compare-sync.md [hunk 35]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
+4. An additional inputs that appears depending on the source type:
+  - For `Cluster`, this input will be a dropdown for selecting the desired context
```
Comment (jfermi) [inline docs/compare-sync.md:64]:
```suggestion
  - For `Cluster`, this input will be a dropdown for selecting the desired context.
```

File: docs/compare-sync.md [hunk 36]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
+  - For `Helm Preview`, there will be two inputs, one for choosing the Chart and one for choosing the Values file
```
Comment (jfermi) [inline docs/compare-sync.md:65]:
```suggestion
  - For `Helm Preview`, there will be two inputs, one for choosing the Chart and one for choosing the Values file.
```

File: docs/compare-sync.md [hunk 37]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
+  - For `Kustomize Preview` there will be one input for choosing the kustomization file
```
Comment (jfermi) [inline docs/compare-sync.md:66]:
```suggestion
  - For `Kustomize Preview` there will be one input for choosing the Kustomization file.
```

File: docs/compare-sync.md [hunk 38]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
...
+
+After both columns are configured with the desired sources, the comparison will take place and you will be able to see the two sets of resources side by side (yellow color in the Overview image).
+
+Clicking on a resource name will open a read-only editor for a quick view of the resource content.
+
+The sections highlighted in light blue in the Overview image show the actions that can be done on the resources.
+In the middle blue section, each row of matching resources have actions that can be currently applied to the resources.
```
Comment (jfermi) [inline docs/compare-sync.md:73]:
```suggestion
In the middle blue section, each row of matching resources have the actions that can be applied to the resources.
```

File: docs/compare-sync.md [hunk 39]
```diff
@@ -30,14 +30,46 @@ To use Compare & Sync follow these steps:
 
 ![Compare & Sync](img/cluster-compare-button-1.6.0.png)
 
-The Compare & Sync modal opens and shows the local resources on the left and the cluster resources on the right. 
+In the image below you can see an overview of the Compare & Sync modal with each section highligthed in color which will be explained:
```
Comment (jfermi) [inline docs/compare-sync.md:33]:
```suggestion
In the image below you can see an overview of the Compare & Sync modal with each section highlighted in a color that will be explained:
```

**Review by jfermi — COMMENTED** [review]
Some editing suggestions.

**Review by topliceanurazvan — APPROVED** [review]

**Pull request merged**